renaissance-core/src/main/java/org/renaissance/RenaissanceBenchmark.java

```diff
@@ -96,6 +96,9 @@ public final Throwable runBenchmark(Config config) {
       }
       return null;
     } catch (Throwable t) {
+      for (ResultObserver observer : config.resultObservers()) {
+        observer.onFailure(name());
+      }
```

Comment:
What would be the consequences of placing this callback in the `finally` block after the `tearDownAfterAll` call?

Reply:
It should be there too :-). Fixed.